tools/htslib/compile.sh

```diff
@@ -4,7 +4,7 @@
 sudo apt-get install -y zlib1g-dev libbz2-dev libcurl4-gnutls-dev libssl-dev autoconf
 
 # Compile LZMA to WebAssembly
-LZMA_VERSION="5.2.5"
+LZMA_VERSION="5.6.3"
```

Comment:
Opportunistic update of LZMA. I don't know what the best tradeoff here is: keep the old one with its problems, or attempt a new one that is untested in this wasm context.